serverless/pages/spaces.mdx

```diff
@@ -0,0 +1,55 @@
+---
+slug: /serverless/spaces
+title: Spaces
+description: Organize your project and objects into multiple spaces
+---
+
+Spaces enable you to organize your dashboards and other saved
+objects into meaningful categories. Once inside a space, you see only
+the dashboards and saved objects that belong to that space.
+
+When you create and enter a new project, you're using the default space of that project.
+
+You can identify the space you're in or switch to a different space from the header.
+
+<DocImage size="l" url="../images/space-breadcrumb.png" alt="Space breadcrumb" />
+
+You can view and manage the spaces of a project from the **Spaces** page in **Management**.
+
+## Required permissions
+
+You must have the admin role on the project to manage its **Spaces**.
+
+## Create or edit a space
+
+You can create as many spaces as you need. 
+
+1. Click *Create space* or select the space you want to edit.
+
+2. Provide:
+    - A meaningful name and description for the space.
+    - A URL identifier. The URL identifier is a short text string that becomes part of the ((kib)) URL. ((kib)) suggests a URL identifier based on the name of your space, but you can customize the identifier to your liking. You cannot change the space identifier later.
+
+3. Customize the avatar of the space to your liking.
+
+4. Save the space.
+
+((kib)) also has an [API](https://www.elastic.co/docs/api/doc/serverless)
```

Comment:
note: APIs will not be available until https://github.com/elastic/kibana/pull/193534 merges

Reply:
I'll either comment this out or update with the correct link when we get more information about publishing the corresponding the API docs

Reply:
Updated with a more precise link now that the API docs are published https://www.elastic.co/docs/api/doc/serverless/group/endpoint-spaces